ApiReview/API.Polly.Core/NoDocs/Polly.Timeout/TimeoutRejectedException.cs

```diff
@@ -0,0 +1,17 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Polly.Timeout;
+
+public class TimeoutRejectedException : ExecutionRejectedException
+{
+    public TimeSpan Timeout { get; }
```

Comment:
What does this return when the constructors that don't take a timeout value are used? In fact, why are there construtor overloads that don't take a timeout?

Reply:
If the no-timeout constructors make sense, then maybe this should be a TimeSpan?

Reply:
This class is type-forwarded to share with the existing implementation in v7, so it's not really new.